src/AksArc/AksArc.Autorest/custom/Get-AzAksArcCluster.ps1

```diff
@@ -0,0 +1,131 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Gets the provisioned cluster instance
+.Description
+Gets the provisioned cluster instance
+.Example
+{{ Add code here }}
+.Example
+{{ Add code here }}
+
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.AksArc.Models.IProvisionedCluster
+.Link
+https://learn.microsoft.com/powershell/module/az.aksarc/get-azaksarcprovisionedclusterinstance
```

Comment:
The online versions in help markdown come from the `Link` session here in custom files. Please correct the links as well as the Synopsis/ Description/ Example/ Outputs for each cmdlet.